### DISCUSSION TITLE
Welcome to BabylonJS-React-Template Discussions!

### DISCUSSION THREAD

**Discussion by canxerian** [microsoft/BabylonJS-React-Template#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Shivamkumar31** [comment]
👋 Hi everyone! I'm Shivam kumar, a beginner exploring React and BabylonJS. I'm passionate about frontend development and 3D graphics.

I'm here to learn and contribute by improving documentation, fixing beginner-friendly issues, or testing features. I’d appreciate any guidance on how to get started.

Looking forward to growing with this awesome community! 🚀
